libs/medic-users-meta/migrations/202402183030.do.telemetry_metrics.sql

```diff
@@ -0,0 +1,37 @@
+DROP MATERIALIZED VIEW IF EXISTS useview_telemetry_metrics CASCADE;
```

Comment:
this cascade delete can cause some real problems... we did this before and the problem is that you never know what you're going to cascade drop. who knows what people have built on these?

is it possible to alter?